packages/venia-concept/src/components/MiniCart/cartOptions.js

```diff
@@ -1,77 +1,71 @@
-import React, { Component, Suspense } from 'react';
+import React, { Suspense, useCallback, useState } from 'react';
 import { array, bool, func, number, shape, string } from 'prop-types';
 import { Form } from 'informed';
 import { Price } from '@magento/peregrine';
 
-import LoadingIndicator from 'src/components/LoadingIndicator';
-import classify from 'src/classify';
-import defaultClasses from './cartOptions.css';
+import { mergeClasses } from 'src/classify';
+import LoadingIndicator, {
+    loadingIndicator
+} from 'src/components/LoadingIndicator';
 import Button from 'src/components/Button';
 import Quantity from 'src/components/ProductQuantity';
 import appendOptionsToPayload from 'src/util/appendOptionsToPayload';
 import isProductConfigurable from 'src/util/isProductConfigurable';
 
+import defaultClasses from './cartOptions.css';
+
 // TODO: get real currencyCode for cartItem
```

Comment:
Resolved in https://github.com/magento-research/pwa-studio/pull/1266/commits/38d0d217b0f270f093c4249adcda5a9fb939b20e.

Because `cart` wasn't passed to `cartOptions` I ended up adding the new `getCurrencyCode` utility function to `Body` and drill it down to `EditItem` and then `CartOptions`.

The prop drilling part isn't ideal but I thought this was better than passing the entire `cart` down.